README.md

```diff
@@ -5,11 +5,10 @@
 </p>
 
 <p align="center">
-A minimalistic Flutter game engine.
+A Flutter based game engine.
```

Comment:
`Flutter-based`

Reply:
Updated, we'd like to change this tagline later, but not enough time today.